robottelo/utils/virtwho.py

```diff
@@ -535,3 +562,35 @@ def get_configure_command_option(deploy_type, args, org=DEFAULT_ORG):
     if deploy_type == 'name':
         return f"hammer -u {username} -p {password} virt-who-config deploy --name {args['name']} --organization '{org}' "
     return None
+
+
+def vw_fake_conf_create(
+    owner,
+    rhsm_hostname,
+    rhsm_username,
+    rhsm_encrypted_password,
+    fake_conf_file,
+    json_file,
+    is_hypervisor=True,
+):
+    conf_name = fake_conf_file.split("/")[-1].split(".")[0]
+    file = f'{fake_conf_file}\n'
+    title = f'[{conf_name}]\n'
+    type = 'type=fake\n'
+    json = f'file={json_file}\n'
+    is_hypervisor = f'is_hypervisor={is_hypervisor}\n'
+    owner = f'owner={owner}\n'
+    env = 'env = Library\n'
+    rhsm_hostname = f'rhsm_hostname={rhsm_hostname}\n'
+    rhsm_username = f'rhsm_username={rhsm_username}\n'
+    rhsm_encrypted_password = f'rhsm_encrypted_password={rhsm_encrypted_password}\n'
+    rhsm_prefix = 'rhsm_prefix=/rhsm\n'
+    rhsm_port = 'rhsm_port=443\n'
+    cmd = f'cat <<EOF > {file}\n{title}\n{type}\n{json}\n{is_hypervisor}\n{owner}\n{env}\n{rhsm_hostname}\n{rhsm_username}\n{rhsm_encrypted_password}\n{rhsm_prefix}\n{rhsm_port}\nEOF'
+    runcmd(cmd)
+
+
+def vw_run_option(option):
+    runcmd("systemctl stop virt-who")
+    runcmd("pkill -9 virt-who")
+    runcmd(f"virt-who -{option}")
```

Comment:
Please use single quotes wherever possible, to ensure consistency :)

Reply:
@Gauravtalreja1  Thank you very much for your suggestion, have already updated